Code/Framework/AzToolsFramework/AzToolsFramework/Entity/EditorEntityContextComponent.cpp

```diff
@@ -284,9 +271,11 @@ namespace AzToolsFramework
         {
             return;
         }
+
         SetupEditorEntity(entity);
```

Comment:
In all the 3 cases (PrefabPublicHandler::CreateEntity, EditorEntityContextComponent::CreateNewEditorEntity, EditorEntityContextComponent::CreateNewEditorEntity), SetupEditorEntity() will be called indirectly through HandleEntitiesAdded(). 
So, you will be doing the same operations twice for prefabs.
So, this line can also be put under if (m_isLegacySliceService)

Reply:
Fixed.